tests/CountArgumentsWrapperTest.php

```diff
@@ -0,0 +1,128 @@
+<?php declare(strict_types=1);
+/**
+ * Php version 8.1.9
+ * 
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
+ */
+use GuzzleHttp\Psr7\Message;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * CountArgumentsTest The class whitch runs
+ * functions/Functions.php/CountArgumentsWrapper and tests it's behavior
+ *
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
+ */
+class CountArgumentsWrapperTest extends TestCase
+{
+    /**
+     * Stores object of instanted class
+     */
+    protected $functions;
+
+    /**
+     * SetUp() Makes an instance of a class
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        $this->functions = new functions\Functions();
+    }
+        
+    /**
+     * TearDown - Destroys instanted class objects
+     *
+     * @return void
+     */
+    protected function tearDown(): void
+    {
+        unset($this->functions);
+    }
+
+    /**
+     * Makes test runs of tested functions and controls it's exception
+     * 
+     * @param $input takes test value from dataprovider? array of function's params
+     * 
+     * @return void
+     * 
+     * @dataProvider negativeDataProvider
+     */
+    public function testCountArgumentsWrapperArrayOfdifferentTypes($input)
+    {
+        try{
+            $this->functions->CountArgumentsWrapper(...$input);
+            $this->fail('An expected exception not thrown');
+        } catch (InvalidArgumentException $exception){
+            $this->assertStringContainsString(
+                'function accepts strings only',
+                $exception->getMessage()
+            );
+        }
+    } 
+    
+    /**
+     * NegativeDataProvider provides pairs of ingoing / outgoing arrays 
+     * for negative tests for exceptions throwing
+     *
+     * @return array 
+     */
+    public function negativeDataProvider(): array
+    {
+        return [
+            [['1','2',3,'4','5']], 
```

Comment:
please add spaces after each ,
and use single quotes ''

Reply:
Done